### ISSUE TITLE
Humdrum -> MEI conversion: sometimes mRests are not written (just empty layers)

### ISSUE THREAD

**Issue by gregchapman-dev** [rism-digital/verovio#3093]
**Describe the problem**
Sometimes a measure-long humdrum rest turns into an empty layer instead of a layer with an mRest in it.

**To Reproduce**
1. Load the Humdrum file attached (measureRest.krn)
2. Convert to MEI (see attached measureRest.mei)
3. See that the first measure has mRests where the Humdrum file has rests that are the same duration as the measure, but the second measure just has empty layers where mRests should be.
Here is an example from measureRest.mei of an empty layer that is missing an mRest:

```
<staff xml:id="staff-L30F11N1" n="7">
     <layer xml:id="layer-L30F11N1" n="1" />
</staff>
```

**Expected behavior**
I expected mRests in the layers that are empty in the second measure.  My current theory is that the time signature change between the two measures exposes this bug.

**Input data**
[measureRest.krn.zip](https://github.com/rism-digital/verovio/files/9873167/measureRest.krn.zip)

**Bad output data**
[measureRest.mei.zip](https://github.com/rism-digital/verovio/files/9873182/measureRest.mei.zip)

**Verovio information**
 - Version of Verovio: develop, version 3.13.0-dev-e2eebd6
 - Binding: command line

**Environment information (as appropriate)**
 - OS: macOS 12.6

**Comment by gregchapman-dev** [comment]
Probable dupe of issue #679, whose fix is waiting in PR #3082.

**Comment by gregchapman-dev** [comment]
Fixed in PR #3082 (I tried it out).
